dp/drivers/i40e.c

```diff
@@ -158,7 +158,7 @@ static void dev_infos_get(struct ix_rte_eth_dev *dev, struct ix_rte_eth_dev_info
 	struct rte_eth_dev_info dpdk_dev_info;
 
 	rte_eth_dev_info_get(dev->port, &dpdk_dev_info);
-	dev_info->nb_rx_fgs = dpdk_dev_info.reta_size;
```

Comment:
Check out this: https://github.com/ix-project/ix/blob/bd4dd0ba200413716071561a3c3f2994159c665d/inc/ix/ethfg.h#L38

There is an upper limit on the number of flow groups that IX supports. Try to raise it,